### ISSUE TITLE
Indicator does not animate between items in v2.0.1

### ISSUE THREAD

**Issue by gapl** [rechsteiner/Parchment#422]
We've upgraded from v1.7.0 to v2.0.1 and followed migration instructions under [v2.0.0 release](https://github.com/rechsteiner/Parchment/releases/tag/v2.0.0).

After changing our instances of `FixedPagingViewController` to non-generic version of `PagingViewController`, indicator does not transition smoothly in between pages.

We're using 2 view controllers in the `PagingViewController`. Here are our styling parameters:

```
pagingController.menuItemSize = .fixed(width: 100, height: 40)
pagingController.borderOptions = .hidden
pagingController.indicatorColor = .purple
pagingController.collectionView.backgroundColor = .almostBlack
pagingController.font = TextStyle.heading4.font
pagingController.selectedFont = TextStyle.heading4.font
pagingController.textColor = .steelGrey
pagingController.selectedTextColor = .silver

pagingController.indicatorOptions = .visible(
    height: 2,
    zIndex: Int.max,
    spacing: UIEdgeInsets(top: 0, left: <computed_value_based_on_text_width>, bottom: 0, right: <computed_value_based_on_text_width>),
    insets: .zero)
```

**Comment by tiennv157** [comment]
Hi @gapl 
Have you fixed this bug? I got a problem like your issue.

**Comment by gapl** [comment]
No, still waiting for response.

**Comment by rechsteiner** [comment]
Hi! I’m away on vacation the next two weeks, so I won’t be able to look into this right now.

Are you seeing this issue in any of the example projects as well?

**Comment by tiennv157** [comment]
Hi, You can check in MultipleCellsExample with 2 or 3 items.
I think problem in sizeDelegate, after calcultor cell width, the anmation
not work

Vào Th 5, 20 thg 2, 2020 lúc 21:52 Martin Rechsteiner <
notifications@github.com> đã viết:

**Comment by rechsteiner** [comment]
Okay, thanks. I’ll get this fixed as soon as I’m back.

**Comment by tiennv157** [comment]
Thank you very much.

Vào Th 5, 20 thg 2, 2020 lúc 23:24 Martin Rechsteiner <
notifications@github.com> đã viết:

**Comment by rechsteiner** [comment]
This should be fixed now (#441) 🙂 I need to do a bit more testing before releasing a new version, but you can target the `master` branch for now. Let me know if you're still seeing any issues!
